lms/djangoapps/mobile_api/video_outlines/tests.py

```diff
@@ -206,6 +216,42 @@ def test_course_list_with_nameless_unit(self):
         self.assertEqual(len(course_outline), 1)
         self.assertEqual(course_outline[0]['path'][2]['name'], self.nameless_unit.location.block_id)
 
+    def test_course_list_with_video_in_sub_section(self):
+        """
+        Tests a non standard xml format where a video is underneath a sequential
+
+        We are expecting to return the same unit and section url since there is
+        no unit vertical.
+        """
+        ItemFactory.create(
+            parent_location=self.sub_section.location,
+            category="video",
+            edx_video_id=self.edx_video_id,
+            display_name=u"test draft video omega 2 \u03a9"
+        )
+        course_outline = self._get_video_summary_list()
+        self.assertEqual(len(course_outline), 1)
+        self.assertEqual(len(course_outline[0]['path']), 2)
+        self.assertEqual(course_outline[0]["section_url"], course_outline[0]["unit_url"])
```

Comment:
These should be equal because the unit returned is only as good as the section_url since the unit position cannot be found.